VideoLocker/src/main/java/org/edx/mobile/model/api/CourseEntry.java

```diff
@@ -128,11 +129,18 @@ public void setNumber(String number) {
         this.number = number;
     }
 
+    public String getDiscussionUrl(){
+        return discussion_url;
+    }
```

Comment:
These new methods deviate from the existing pattern in this model class, which is to define getter and setter methods in one line.